### PR TITLE
Modal translation

### PR DESCRIPTION
**Pull request by lixiaoyang1992**

### CODE REVIEW

File: components/modal/demo/alert.md [hunk 1]
```diff
@@ -5,8 +5,14 @@ title:
   en-US: alert
 ---
 
+## zh-CN
+
 包含无按钮, 确认框, 多按钮情况
 
+## en-US
+
+including situations like no button, confirm button and multi button
```
Comment (afc163) [inline components/modal/demo/alert.md]: including => Including

multi button => multiple buttons

including situations like => Including

File: components/modal/demo/basic.md [hunk 2]
```diff
@@ -5,8 +5,14 @@ title:
   en-US: Basic
 ---
 
+## zh-CN
+
 基本对话框
 
+## en-US
+
+Basic Modal
```
Comment (afc163) [inline components/modal/demo/basic.md]: Always need punctuation.

File: components/modal/demo/operation.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
   en-US: operation
 ---
 
+## zh-CN
+
 操作对话框
 
+## en-US
+
+operation modal
```
Comment (afc163) [inline components/modal/demo/operation.md]: Modal with operations.

File: components/modal/index.en-US.md [hunk 4]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
-
-### 规则
-- 尽可能少用。Modal 会打断用户操作，只用在重要的时候。
-- 标题应该简明，不能超过 1 行；描述内容应该简明、完整，一般不多于 2 行。
-- 操作按钮最多到 3 个（竖排），一般为 1-2 个（横排）；3 个以上建议使用组件 ActionSheet 来完成。
-- 一般将用户最可能点击的按钮，放在右侧。另外，取消按钮应当始终放在左侧。
+Use to show important information for the system, and request the user to operate feedback. eg: When deleting an important content, pop up a Moal for second confirmation
 
+### Rules
+- Use as few as possible. Modal will interrupt user operation, only use it at important time.
```
Comment (afc163) [inline components/modal/index.en-US.md]: time => situation

File: components/modal/index.en-US.md [hunk 5]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
 ---
 
-用作显示系统的重要信息，并请求用户进行操作反馈，eg：删除某个重要内容时，弹出 Modal 进行二次确认。
-
-### 规则
-- 尽可能少用。Modal 会打断用户操作，只用在重要的时候。
-- 标题应该简明，不能超过 1 行；描述内容应该简明、完整，一般不多于 2 行。
-- 操作按钮最多到 3 个（竖排），一般为 1-2 个（横排）；3 个以上建议使用组件 ActionSheet 来完成。
-- 一般将用户最可能点击的按钮，放在右侧。另外，取消按钮应当始终放在左侧。
+Use to show important information for the system, and request the user to operate feedback. eg: When deleting an important content, pop up a Moal for second confirmation
```
Comment (afc163) [inline components/modal/index.en-US.md]: `request the user to operate feedback` => `ask for user feeback`

Comment (afc163) [inline components/modal/index.en-US.md]: Moal => MoDal

File: components/modal/index.en-US.md [hunk 6]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
 
+### Rules
+- Use as few as possible. Modal will interrupt user operation, only use it at important time.
+- Title should be concise, can't exceed 1 line; description should be concise and complete, generally no more than 2 lines.
+- Operation buttons are up to 3(vertical), generally 1-2(horizontal); more than 3 is suggested to use component ActionSheet.
```
Comment (afc163) [inline components/modal/index.en-US.md]:
```diff
- more than 3 is suggested to use component ActionSheet.
+ [ActionSheet](/components/action-sheet) is prefered when there are more than 3 actions.
```

**Comment by paranoidjk** [comment]
- 有个原则，不用太依赖原来的中文文档逐字逐词翻译，尽量意译，尽量符合英文的顺序习惯
- btw，原来的中文文档可能也不是非常完美，接下一个组件的翻译任务非常欢迎同时把中文文档也调整优化下，一举两得

File: components/modal/demo/propmt.md [hunk 7]
```diff
@@ -5,7 +5,13 @@ title:
...
 ---
 
-包含输入普通文字, 密码, 登录信息样式
+## zh-CN
+
+包含输入普通文字, 密码, 登录信息样式。
+
+## en-US
+
+Including styles for typing in normal word, password and sign in information.
```
Comment (paranoidjk) [inline components/modal/demo/propmt.md]: Including normal text input modal, password input modal and login information input modal.

File: components/modal/index.en-US.md [hunk 8]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
 ---
 
-用作显示系统的重要信息，并请求用户进行操作反馈，eg：删除某个重要内容时，弹出 Modal 进行二次确认。
-
-### 规则
-- 尽可能少用。Modal 会打断用户操作，只用在重要的时候。
-- 标题应该简明，不能超过 1 行；描述内容应该简明、完整，一般不多于 2 行。
-- 操作按钮最多到 3 个（竖排），一般为 1-2 个（横排）；3 个以上建议使用组件 ActionSheet 来完成。
-- 一般将用户最可能点击的按钮，放在右侧。另外，取消按钮应当始终放在左侧。
+Use to show important information for the system, and ask for user feedback. eg: When deleting an important content, pop up a Modal for second confirmation.
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: Secondary confirmation

Comment (afc163) [inline components/modal/index.en-US.md]: Secondary is wrong.

Comment (paranoidjk) [inline components/modal/index.en-US.md]: feels ok, 

https://www.google.com/search?newwindow=1&safe=strict&hl=zh-CN&site=webhp&source=hp&q=Secondary+confirmation&btnG=Google+%E6%90%9C%E7%B4%A2&oq=&gs_l=
or use `the second time confirm`

File: components/modal/index.en-US.md [hunk 9]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
+### Rules
+- Use as few as possible. Modal will interrupt user operation, only use it at important situation.
+- Title should be concise, can't exceed 1 line; description should be concise and complete, generally no more than 2 lines.
+- Operation buttons are up to 3(vertical), generally 1-2(horizontal); [ActionSheet](/components/action-sheet) is preferred when there are more than 3 actions.
+- Generally put the most likely clicked button on the right side. In addition, the cancel button should always be on the left.
 
 ## API
 
-### Modal ( 适用平台：WEB、React-Native )
+### Modal ( Support PlatForm：WEB、React-Native )
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: Support Platform

File: components/modal/index.en-US.md [hunk 10]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
-| maskClosable (only transparent) | 点击蒙层是否允许关闭 | Boolean   | true       |
-| footer  (only not transparent)     | 底部内容       |  Array [{text, onPress}]    | [] |
-| transparent | 是否弹窗模式       | Boolean   |  false |
-| animationType (`rn only`) | 可选: 'slide-down/up'(transparent 模式下) / 'fade' / 'slide'(仅非 tranparent) | String |   fade |
-| style (`web only`) |  样式    | Object | 透明模式下: {width: '286px', height: 'auto'}, <br />非透明模式:  {width: '100%', height: '100%'} (web)|
-
-### Modal.alert(title, message, actions?) ( 适用平台：WEB、React-Native )
-
-| 参数             | 说明                                         | 类型     | 默认值        |
+| prefixCls (web only)      | Style class name prefix |    String   | `am-modal`      |
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: ~~Style class name prefix~~ --> class name prefix of elements

File: components/modal/index.en-US.md [hunk 11]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
 
-| 参数             | 说明                                         | 类型     | 默认值        |
+| Property             | Description                                         | Type     | Default        |
 |------------------|----------------------------------------------|----------|---------------|
-| title        | 标题                      | String 或 React.Element   | 无            |
-| message      | 提示信息                  | String 或 React.Element                    | 无    |
-| callbackOrActions  | 按钮组 [{text, onPress}] 或回调函数      | Array or Function | 无            |
-| type       | prompt 的样式   | String (`default`, `secure-text`, `login-password`)|  `default`  |
-| defaultValue       | 默认值(input 为 password 类型不支持)   | String |   -  |
+| actions         | button group, [{text, onPress, style}]       | Array | no            |
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: ~~no~~, maybe use `-`, like ant design documentation

File: components/modal/index.en-US.md [hunk 12]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
-| transparent | 是否弹窗模式       | Boolean   |  false |
-| animationType (`rn only`) | 可选: 'slide-down/up'(transparent 模式下) / 'fade' / 'slide'(仅非 tranparent) | String |   fade |
-| style (`web only`) |  样式    | Object | 透明模式下: {width: '286px', height: 'auto'}, <br />非透明模式:  {width: '100%', height: '100%'} (web)|
-
-### Modal.alert(title, message, actions?) ( 适用平台：WEB、React-Native )
-
-| 参数             | 说明                                         | 类型     | 默认值        |
+| prefixCls (web only)      | Style class name prefix |    String   | `am-modal`      |
+| visible      | Determine whether a modal dialog is visible or not | Boolean          | false           |
+| onClose      | Callback for clicking x or mask        | (): void   | no |
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: ~~x~~ --> close icon x

File: components/modal/index.en-US.md [hunk 13]
```diff
@@ -4,46 +4,51 @@ type: Feedback
...
+| title (only transparent)       | title           | React.Element    | no           |
+| closable    | Determine whether a close (x) button is visible or not | Boolean    | true        |
```
Comment (paranoidjk) [inline components/modal/index.en-US.md]: ~~false~~ --> `false`
I think it's better to use  ``  wrap defaultValue :)

**Comment by paranoidjk** [comment]
@lixiaoyang1992 rebase -i 合并一下commit？ github sqush出来会保留全部commit信息不太好看

辛苦了 👍

**Pull request merged**